src/Symfony/Bundle/FrameworkBundle/Tests/Console/ApplicationTest.php

```diff
@@ -48,7 +94,7 @@ public function testBundleCommandsAreRegistered()
         $application->doRun(new ArrayInput(array('list')), new NullOutput());
     }
 
-    public function testBundleCommandsAreRetrievable()
+    public function testBundleCommandsAreRetrievable(): void
```

Comment:
the added typehints on existing tests should all be reverted as they would cause merge conflicts with no additional value.